### ISSUE TITLE
Audio clipping

### ISSUE THREAD

**Issue by mmontag** [kmturley/sfz-web-player#6]
Hi, I'm sorry to be harsh, but the sound clips so easily that it's almost unusable. 
Can you add a gain control and significantly reduce the default output level?

**Comment by kmturley** [comment]
Hello, it is not harsh to provide valuable feedback.
Can you provide more information about the machine/browser you are using and which instrument so I can try to replicate?

This sfz-web-player is a prototype so does not have many audio effects/controls implemented yet

Here is a video showing notes played with a volume indicator. Looks ok to me at first glance:
https://github.com/sfzlab/sfz-web-player/assets/1710493/657daf46-bbb3-4ef7-b850-03754678afeb

It would be fairly easy to add a volume control if using your overall machine volume is not sufficient

**Comment by mmontag** [comment]
I don't think it is machine/browser specific, but I could be wrong. I think the engine is internally clipping. (Turning down the system volume does not alleviate the clipping)

I am just loading up the page with the default instrument, which sounds like Kalimba, and then clicking my mouse across all the notes. As soon as 3-4 notes are playing simultaneously, it sounds like clipping to me.

**Comment by kmturley** [comment]
Ah yes I can replicate. Playing multiple notes together does cause distortion/clipping.
Will add as a bug to fix

**Comment by missinglink** [comment]
Hi @kmturley, did you manage to fix this, or would you be interested in accepting a PR for it if you could pout out where the error resides?

**Comment by kmturley** [comment]
Hello, I didn't take a look at it yet as I've spent my time on my Open Audio Stack project.

Definitely open to contributions. Currently there is no audio engine logic, start stop/loops etc. I think this could be really useful if that was implemented!
